### ISSUE TITLE
Clarification about JSON formats

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#1029]
Hi @VibhorCodecianGupta just wondering about these two sections in the readme and any ambiguities about them:

https://github.com/publiclab/image-sequencer/blob/main/README.md#adding-steps-to-the-image

mentions using JSON and offers an example. And i was wondering how to pass in options in the JSON format, so perhaps another example of that would be great (even including a function for a module that accepts that, like blend?)

But then this section lower down references JSON format too, but it seems more complex:

https://github.com/publiclab/image-sequencer/blob/main/README.md#importing-steps-using-json-array

Any thoughts on these? Thanks and sorry i'm just trying to read this as a newcomer might. Thank you!

**Comment by VibhorCodecianGupta** [comment]
@jywarren you're right, maybe we can document this better.

I mentioned [here](https://github.com/publiclab/image-sequencer/blob/main/README.md#adding-steps-to-the-image) that IS supports string syntax and JSON. The syntax `brightness{brightness:80}` is actually closer to a stringified JSON. Perhaps we can specify that the new support allows for string syntax which includes stringified JSON? This is because JSON is already supported [here](https://github.com/publiclab/image-sequencer/blob/main/README.md#importing-steps-using-json-array) in raw  form, and docs seem redundant this way. Thoughts?

**Comment by jywarren** [comment]
I guess perhaps we should think about running /everything/ through the same auto-identifier of syntax, and not having different ways JSON works. The string syntax is, indeed, intended to be similar to simplified JSON, but we skipped `:` and `,` to make it more compact in URLs. 

I think the main thing is we should provide more full examples of the JSON that's accepted. Then we can look at where and if they differ, and how we could consolidate and standardize it. What do you think?

**Comment by VibhorCodecianGupta** [comment]
@jywarren agreed. So this is first a documentation issue and then perhaps a refactoring issue if need be. I'll get on to it in some time, have been caught up due to my exams coming up 😅 I also made some suggestions to your concerns in my proposal, if you could have a look here 
https://publiclab.org/notes/vibhordelgupta/04-09-2019/soc-proposal-image-sequencer-broadening-avenues?_=1555858027
